data/hails.txt

```diff
@@ -8,6 +8,17 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+phrase "passenger"
+	word
+		"rich kid"
+		"spoiled brat"
+		"salesman"
+		"fancy lady"
+		"old man"
+		"retired cop"
+		"college student"
+		"ex-military"
```

Comment:
This one doesn't work because it's an adjective, not a noun. (e.g. "I just gave a lift to this ex-military")